src/Nethermind/Nethermind.Core/Account.cs

```diff
@@ -8,7 +8,7 @@ namespace Nethermind.Core
 {
     public class Account
```

Comment:
Ah still exist?

Reply:
There are places where we need to put it on heap, not sure if worth paying for boxing?

Reply:
But will try to move some usages to AccountStruct in next PR.